lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_send_app_size_metrics.rb

```diff
@@ -0,0 +1,165 @@
+            (1) Use either `ad-hoc`, `enterprise` or `development` export method (in particular, won't work with `app-store`),
+            (2) Provide `thinning: '<thin-for-all-variants>'` as part of your `export_options` of `gym` (or in your `options.plist` file if you use raw `xcodebuild`)
+          See https://help.apple.com/xcode/mac/11.0/index.html#/devde46df08a
+
+          For builds exported with the `app-store` method, `xcodebuild` won't generate an `app-thinning.plist` file; so you will only be able to get
+          the Universal `.ipa` file size as a metric, but won't get the per-device, broken-down install and download sizes for each thinned variant.
+
+          See https://github.com/Automattic/apps-metrics for the API contract expected by the Metrics server you are expected to send those metrics to.
+
+          Tip: If you provide a `file://` URL for the `api_url`, the action will write the payload on disk at the specified path instead of sending
+          the data to a endpoint over network. This can be useful e.g. to inspect the payload and debug it, or to store the metrics data as CI artefacts.
+        DETAILS
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(
+            key: :api_url,
+            env_name: 'FL_IOS_SEND_APP_SIZE_METRICS_API_URL',
+            description: 'The endpoint API URL to publish metrics to. (Note: you can also point to a `file://` URL to write the payload to a file instead)',
+            type: String,
+            optional: false
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :api_token,
+            env_name: 'FL_IOS_SEND_APP_SIZE_METRICS_API_TOKEN',
+            description: 'The bearer token to call the API. Required, unless `api_url` is a `file://` URL',
+            type: String,
+            optional: true
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :use_gzip_content_encoding,
+            env_name: 'FL_IOS_SEND_APP_SIZE_METRICS_USE_GZIP_CONTENT_ENCODING',
+            description: 'Specify that we should use `Content-Encoding: gzip` and gzip the body when sending the request',
+            type: FastlaneCore::Boolean,
+            default_value: true
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :app_name,
+            env_name: 'FL_IOS_SEND_APP_SIZE_METRICS_APP_NAME',
+            description: 'The name of the app for which we are publishing metrics, to help filter by app in the dashboard',
+            type: String,
+            optional: false
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :app_version,
+            env_name: 'FL_IOS_SEND_APP_SIZE_METRICS_APP_VERSION',
+            description: 'The version of the app for which we are publishing metrics, to help filter by version in the dashboard',
```

Comment:
Nitpick: There might use cases other than a dashboard where we want to filter or group by version:

```suggestion
            description: 'The version of the app for which we are publishing metrics, to help with filtering and grouping',
```